### ISSUE TITLE
Questions about imputation in TimesNet

### ISSUE THREAD

**Issue by liyiersan** [thuml/Time-Series-Library#356]
https://github.com/thuml/Time-Series-Library/blob/8782a590950b4ee31a0d2d66fc30b761689c3636/models/TimesNet.py#L131C1-L140C1
would the codes be better by adding an epsilon to avoid being divided by 0 as follows:
```python
        # Normalization from Non-stationary Transformer
        valid_num = torch.sum(mask == 1, dim=1) + 1e-5
        means = torch.sum(x_enc, dim=1) / valid_num
        means = means.unsqueeze(1).detach()
        x_enc = x_enc - means
        stdev = torch.sqrt(torch.sum(x_enc * x_enc, dim=1) / valid_num) + 1e-5
        stdev = stdev.unsqueeze(1).detach()
        x_enc /= stdev
        
        x_enc = x_enc.masked_fill(mask == 0, 0)
```

**Comment by wuhaixu2016** [comment]
Hi, it is more rigorous to add an epsilon. 
Since the mask is generated by sampling w.r.t. the mask ratio (>0), it is non-zero with a large probability. This code will work well here.
